scrabble.js

```diff
@@ -1,15 +1,135 @@
-
+//A, E, I, O, U, L, N, R, S, T	1
+// D, G	2
+// B, C, M, P	3
+// F, H, V, W, Y	4
+// K	5
+// J, X	8
+// Q, Z	10
 const Scrabble = {
+  scoreBoard: {
+    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+    'D': 2, 'G': 2,
+    'B': 3, 'C': 3, 'M': 3, 'P': 3,
+    'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+    'K': 5,
+    'J': 8, 'X': 8,
+    'Q': 10, 'Z': 10
+  },
+
   score(word) {
+    let totalScore = 0;
+
+    // throws if word contains invalid characters
+    // .search will return -1 if there is a match
+    // throws if word is empty
+    //throws if word is longer then 7 characters
+    if (word.search(/[^a-zA-Z]+/) !== -1 || word.length === 0 || word.length > 7)  {
+      throw 'Invalid word.';
+    }
```

Comment:
Instead of throwing a string, you should throw an instance of `Error`. That gives you access to stack trace information. Something like this:

```javascript
if (condition) {
  throw new Error(`Invalid word ${word}`);
}
```